spec/system/claims/support/claims/view_claims_spec.rb

```diff
@@ -12,7 +12,7 @@
 
   scenario "Support user visits the claims index page" do
     when_i_visit_claim_index_page
-    i_see_a_list_of_claims
+    i_see_a_list_of_submitted_claims
```

Comment:
Can we rename this to follow the convention we have
```suggestion
    then_i_see_a_list_of_submitted_claims
```